tasm-test/src/TasmTest.cc

```diff
@@ -103,3 +103,10 @@ TEST_F(TasmTestFixture, testSelectBird) {
     }
     // assert(count == 360);
 }
+
+TEST_F(TasmTestFixture, testInMemory) {
+    tasm::TASM tasm(true);
+
+    tasm.addMetadata("video", "label", 0, 0, 0, 0, 0);
+    tasm.addMetadata("video", "label", 1, 0, 0, 0, 0);
+}
```

Comment:
I am not completely sure if we have to test the in-memory DB at this level; it may be sufficient to just test in the SemanticIndex tests, but I could be convinced otherwise. What is your goal with this unit test? It's not clear to me how much running `addMetadata` tells us.

Nit: It looks like a newline at the end of the file needs to be added.